src/Controller/RestaurantController.php

```diff
@@ -567,6 +570,7 @@ public function cartAction($id, Request $request,
             $this->orderManager->flush();
         }
 
+        $this->eventBus->handle(new OrderUpdated($cart));
```

Comment:
Would have been cool to avoid duplicating this piece of code, and have it in a central place. Maybe in `OrderModifier`?